package.json

```diff
@@ -5,7 +5,8 @@
   "main": "build/index.js",
   "types": "src/index.ts",
   "scripts": {
-    "precommit": "prettier --use-tabs --single-quote --write \"{src,typings,test}/**/*.ts\"",
+    "precommit": "npm run format && npm run build && git add build",
```

Comment:
I think it should be fine with just `--update`, especially passing the build directory.